pkg/core/interop/runtime/util.go

```diff
@@ -74,3 +77,19 @@ func GetNetwork(ic *interop.Context) error {
 	ic.VM.Estack().PushVal(uint32(m))
 	return nil
 }
+
+// GetRandom returns pseudo-random number which depends on block nonce and transaction hash.
+func GetRandom(ic *interop.Context) error {
```

Comment:
And we also need the corresponding interop API and wrapper for CLI.

Reply:
What do you mean by wrapper for CLI?

Reply:
I thought about our interop package for external users, it's what you did in ` pkg/interop/runtime/runtime.go`, but somehow I wrote `wrapper for CLI`, it's a mistake.

Reply:
So it's OK now.